eslint.config.js

```diff
@@ -2,6 +2,7 @@ import js from '@eslint/js'
 import globals from 'globals'
 import reactHooks from 'eslint-plugin-react-hooks'
 import reactRefresh from 'eslint-plugin-react-refresh'
+import prettierConfig from 'eslint-config-prettier';
```

Comment:
### `eslint-config-prettier`
This configuration is used to **disable any ESLint rules that might conflict with Prettier**.

### `eslint-plugin-prettier`
This plugin runs Prettier as an ESLint rule. Instead of separating linting and formatting, **it allows Prettier to run inside ESLint**, and you’ll see Prettier issues as ESLint errors.
>[!WARNING]
> However, this can affect performance negatively!
> https://typescript-eslint.io/users/what-about-formatting#suggested-usage---prettier
>
> ![image](https://github.com/user-attachments/assets/c85b0c31-4865-4693-8c02-d583a32dbedb)